docs/extensions/fares-v2.md

```diff
@@ -0,0 +1,82 @@
+<a class="pencil-link" href="https://github.com/MobilityData/gtfs.org/edit/main/docs/extensions/fare-extension.md" title="Edit this page" target="_blank">
+    <svg class="pencil" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M10 20H6V4h7v5h5v3.1l2-2V8l-6-6H6c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h4v-2m10.2-7c.1 0 .3.1.4.2l1.3 1.3c.2.2.2.6 0 .8l-1 1-2.1-2.1 1-1c.1-.1.2-.2.4-.2m0 3.9L14.1 23H12v-2.1l6.1-6.1 2.1 2.1Z"></path></svg>
+  </a>
+  
+# GTFS-Fares v2
+
+Fares v2 is a GTFS Schedule extension project that aims to address the limitations of <a href="/schedule/examples/fares-v1">Fares v1</a>.
+
+The main concepts that Fares v2 plans to represent are
+
+- Fare products (e.g tickets and passes)
+- Rider categories (e.g seniors and children)
+- Fare payment options (e.g bank card and credit card)
+- Fare capping
+
+These concepts will allow data producers to model zone-based, time-dependent, and inter-agency fares. This extension project is being adopted in iterations. 
+
+Currently, the officially adopted base implementation of Fares v2 works for the following use cases:
+
+- Defining a transit fare
+- Describing service locations in the same fare zone
+- Creating rules for single leg journeys
+- Creating rules for transfers
+
+You can see <a href="/schedule/examples/fares-v2" target="_blank">examples here</a> that show what can be modelled using the official Fares v2 base implementation.
+
+Producers may implement Fares v2 in the same dataset with Fares v1, since there is no technical conflict between the two. Consumers can choose which version to use independent of the other. With adoption and sufficient endorsement of Fares v2, Fares v1 may be deprecated in the future.
+
+<a class="button no-icon" href="https://share.mobilitydata.org/gtfs-fares-v2" target="_blank">See the full proposal</a>
+
+## In Progress Fares v2 Work
+
+The GTFS community is currently working on finalizing the <a href="https://share.mobilitydata.org/fare-containers-to-fare-payment-types-proposal" target="_blank">fare_payment_options.txt (formerly fare_containers.txt)</a> proposal. 
```

Comment:
I think this info is a little duplicative of the table below. From a maintainability standpoint, having only one place to update what we are currently working on might be easier?

Reply:
Hoping the "what we are currently working on" will get updated more often in 2023 🙈

Reply:
The main value of this section to me is having a few sentences that someone can skim for a general update of 

- what feature is being prepared for a vote 
- which proposal they can look at to get up to speed

(and not having to scrounge through GitHub to find that update in a specific PR or issue). 

It serves a different purpose than a list of proposals people have to read through on ProductBoard, since that's more for people who really want to get into the details. I imagine us updating this section every month or two. @isabelle-dr does that make sense to you or is there an alternative approach you'd prefer?

Reply:
Okay I understand better. Sounds good then.
Can you change the name to "fare container updated proposal"? 
The name is back on the table 🙈